scripts/scil_visualize_fodf.py

```diff
@@ -146,6 +146,13 @@ def _build_arg_parser():
                                    help='Length of the peaks segments. '
                                         '[%(default)s]')
 
+    # fODF variance options
+    p.add_argument('--variance',
+                   help='FODF variance (mean + k * variance) file.')
```

Comment:
Here there is some documentation needed. You should explain why k is needed (I believe it is not). 
You could explain that you can use a factor (maybe with scil_image_math.py) for visualization reason or what not.

But this k factor here is not clear.

Reply:
Is this clearer?

Reply:
Put k as a variable. I'm more comfortable to ask for variance file and factor than a file containing k * variance.

Reply:
I changed it!